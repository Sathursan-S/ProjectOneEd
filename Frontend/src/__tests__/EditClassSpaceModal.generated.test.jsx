import renderer from 'react-test-renderer';
import React, { useState } from 'react';

import EditClassSpaceModal from '../Components/EditClassSpaceModal/EditClassSpaceModal';

jest.mock('./EditClassSpaceModal.css');

const renderTree = tree => renderer.create(tree);
describe('<EditClassSpaceModal>', () => {
  it('should render component', () => {
    expect(renderTree(<EditClassSpaceModal 
    />).toJSON()).toMatchSnapshot();
  });
  
});