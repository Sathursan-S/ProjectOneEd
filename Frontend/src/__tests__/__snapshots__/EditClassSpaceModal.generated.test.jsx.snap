// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<EditClassSpaceModal> should render component 1`] = `
<div
  className="modal-backdrop"
>
  <div
    className="modal-content"
  >
    <button
      className="modal-close-button"
    >
      X
    </button>
    <form
      onSubmit={[Function]}
    >
      <h4>
        Edit class space
      </h4>
      <div
        className="form-group"
      >
        <label
          htmlFor="upload-image"
        >
          Change cover image (Optional)
        </label>
        <input
          id="upload-image"
          onChange={[Function]}
          type="file"
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="name"
        >
          Name
        </label>
        <input
          id="name"
          onChange={[Function]}
          required={true}
          type="text"
          value=""
        />
      </div>
      <div
        className="form-group"
      >
        <label
          htmlFor="description"
        >
          Description
        </label>
        <textarea
          id="description"
          onChange={[Function]}
          required={true}
          value=""
        />
      </div>
      <button
        className="submit-button Button"
        type="submit"
      >
        Save changes
      </button>
    </form>
  </div>
</div>
`;
